rubin_scheduler/scheduler/basis_functions/basis_functions.py

```diff
@@ -1208,7 +1234,7 @@ def __init__(
         self.aways_available = aways_available
 
         self.survey_features = {}
-        self.survey_features["Last_observation"] = features.Last_observation()
+        self.survey_features["Last_observation"] = features.LastObservation()
```

Comment:
Looks like `GoalStrictFilterBasisFunction` can get a dep warning or deleted too.

Reply:
Hmm. It should be sending a warning  -- `send_unused_deprecation_warning(self.__class__.__name__)` is at the end of the init.